File: src/PsiCollaborator.Data/Schedule/ScheduleDaily/ScheduleDaily.cs
using System;

namespace PsiCollaborator.Data.Schedule.ScheduleDaily
{
    public class ScheduleDaily : IScheduleDaily
    {
        public int ScheduleDailyId { get; set; }
        public int DayId { get; set; }
        public string DayName { get; set; }
        public DateTime BeginTime { get; set; }
        public DateTime EndTime { get; set; }
    }
}

File: src/PsiCollaboratorManager/Scripts/Lib/Schedule/ScheduleDaily.js
class ScheduleDaily {
    constructor(id, dayId, dayName, startTime, endTime) {
        this.ScheduleDailyId = id;
        this.ScheduleDayId = dayId;
        this.ScheduleDayName = dayName;
        this.BeginTime = startTime;
        this.EndTime = endTime;
        this.TotalTime = endTime.difference(startTime);
    }
}
